### DISCUSSION TITLE
Bulk Actions

### DISCUSSION THREAD

**Discussion by 22syamsudar** [rappasoft/laravel-livewire-tables#411] [Q&A]
I try to dd($this->selectedKeys) but it returns 

 => [
       0=>   1,
       1=>   2,
       2=>   3,
       3=>  4,
 ]   

how to return like below? 
public function exportSelected()
{
    if (count($this->selectedKeys)) {
        // Do something with the selected rows
        dd($this->selectedKeys);
        
//        => [
//            1,
//            2,
//            3,
//            4,
//        ]   
    }

    // Notify there is nothing to export
}

**Comment by KevinYobeth** [comment]
The 0, 1, 2, 3 in front of it is only an index, you can still use it as normal.
